### PR TITLE
Crud with card

### PR DESCRIPTION
**Pull request by VitaliKozhurov**

### CODE REVIEW

File: src/features/cards/ui/addCard/cardForm/useAddForm.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+const addCardSchema = z.object({
+  questionFormat: z.string().trim(),
+  answerFormat: z.string().trim(),
+  question: z.string().min(3, 'Use 3 characters or more for your question').trim(),
+  answer: z.string().min(3, 'Use 3 characters or more for your answer').trim(),
+})
+
+export type CardFormValues = z.infer<typeof addCardSchema>
+type DefaultCardVAlueType = Omit<CardFormValues, 'answerFormat' | 'questionFormat'>
```
Comment (vitalyabedik) [inline src/features/cards/ui/addCard/cardForm/useAddForm.ts:13]: fix name of type A

File: src/features/cards/ui/editCard/EditCard.tsx [hunk 2]
```diff
@@ -1,34 +1,28 @@
...
-        card={card}
-        placeholder={placeholder}
-        options={options}
-        onSubmit={onSubmit}
-        closeModal={closeModal}
-      />
-    </Modal>
+    <EditCardModal
+      trigger={<IconButton icon={<EditIcon />} size={1.6} />}
+      placeholder={'Data format type'}
```
Comment (vitalyabedik) [inline src/features/cards/ui/editCard/EditCard.tsx:19]: without { }

**Review by vitalyabedik — APPROVED** [review]
Good!

File: src/features/cards/ui/editCard/editCardModal/EditCardModal.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+type Story = StoryObj<typeof meta>
+
+const options: OptionType[] = [
+  { value: 'text', title: 'Text' },
+  { value: 'picture', title: 'Picture' },
+]
+
+export const DefaultEditCardModal: Story = {
+  args: {
+    trigger: <IconButton icon={<EditIcon size={1.6} />} />,
```
Comment (DenisKulik) [inline src/features/cards/ui/editCard/editCardModal/EditCardModal.stories.tsx:24]: pass prop size to the IconButton component (not EditIcon)

**Review by DenisKulik — APPROVED** [review]
Good job!

File: src/features/cards/ui/editCard/editCardModal/EditCardModal.stories.tsx (hunk 3 above)
Comment (VitaliKozhurov) [reply src/features/cards/ui/editCard/editCardModal/EditCardModal.stories.tsx:24]: Done!

File: src/features/cards/ui/editCard/EditCard.tsx (hunk 2 above)
Comment (VitaliKozhurov) [reply src/features/cards/ui/editCard/EditCard.tsx]: Done!

File: src/features/cards/ui/addCard/cardForm/useAddForm.ts (hunk 1 above)
Comment (VitaliKozhurov) [reply src/features/cards/ui/addCard/cardForm/useAddForm.ts]: Done!

**Pull request merged**